src/pykiso/lib/connectors/cc_rtt_segger.py

```diff
@@ -286,19 +284,16 @@ def _cc_send(self, msg: Message or bytes, raw: bool = False) -> None:
                 f"ERROR occurred while sending {len(msg)} bytes on buffer {self.tx_buffer_idx}"
             )
 
-    def _cc_receive(
-        self, timeout: float = 0.1, raw: bool = False
-    ) -> Dict[str, Union[Message, bytes, None]]:
+    def _cc_receive(self, timeout: float = 0.1, **kwargs) -> Dict[str, Optional[bytes]]:
```

Comment:
I think you could explicitly specify the size kwarg here and set it to None by default so it appears in the docstring, the code will be clearer IMO

Reply:
Done